src/components/MediaTab.tsx

```diff
@@ -176,9 +176,9 @@ interface IRecordProps {
 
 interface IProps
   extends IStateProps,
-    IDispatchProps,
-    IRecordProps,
-    WithDataProps {
+  IDispatchProps,
```

Comment:
why are these formatting differently on your computer and on mine?

Reply:
sigh...we'll have to zoom to figure it out...

Reply:
I didn't have all the extensions installed on my new local user.  I have them now, so hopefully this won't continue.